mark.php

```diff
@@ -0,0 +1,45 @@
+<?php
+session_start();
+require 'db_connection.php';
+
+$questions_id = $_GET['questions_id'];
+$test_answers = $_GET['test_answers'];
+
+$score = 0.0;
+
+for ($i = 0; $i < count($test_answers); $i ++) {
+    var_dump("STEP 1");
+    for ($j = 0; $j < count($questions_id); $j ++) {
+        var_dump("STEP 2");
+        $sql = 'SELECT * FROM correct_answers WHERE questions_id = :questions_id';
+        $stmt = $db->prepare($sql);
+        $stmt->execute([':questions_id' => $questions_id]);
```

Comment:
$questions_idは配列だと思われるので
```suggestion
        $stmt->execute([':questions_id' => $questions_id[$j]);
```
になると思います。